src/types/accommodation.ts

```diff
@@ -23,3 +25,76 @@ export interface AddressValues {
   latitude: number | null;
   longitude: number | null;
 }
+
+export interface AdressListingValues {
+  street?: string;
+  city?: string;
+  country?: string;
+}
+
+export type OrderOptions = null | 'asc' | 'desc';
+
+export interface SearchValues {
+  location?: AdressListingValues;
+  checkInDate?: string;
+  checkOutDate?: string;
+  orderByPrice?: OrderOptions;
+  orderByRooms?: OrderOptions;
+  orderByPeople?: OrderOptions;
+  minPrice?: number;
+  maxPrice?: number;
+  minRooms?: number;
+  maxRooms?: number;
+  minPeople?: number;
+  maxPeople?: number;
+  limit?: number;
+  page?: number;
+}
+
+export interface ExploreListItem {
+  id: string;
+  thumbnailUrl: string;
+  squareMeters: number;
+  numberOfRooms: number;
+  allowedNumberOfPeople: number;
+  price: number;
+  address: AdressListingValues;
+}
+
+export interface Media {
+  imageUrl: string;
+  thumbnailUrl: string;
+  accommodationId: string;
+}
+
+export interface AccommodationFullView {
```

Comment:
should be `AccommodationDetails` or something like this

Reply:
renamed